app/src/main/java/ch/epfl/sdp/location/LocationUtils.java

```diff
@@ -14,10 +14,8 @@ public static Location buildLocation(double latitude, double longitude) {
         l.setLatitude(latitude);
         l.setLongitude(longitude);
         l.setTime(System.currentTimeMillis());
-        if (Build.VERSION.SDK_INT > Build.VERSION_CODES.JELLY_BEAN) {
```

Comment:
Are you sure we can delete this if ?

I think we target an API above jelly bean so it should be okay

Reply:
Yes